minimesos/src/main/java/com/containersol/minimesos/mesos/MesosClusterContainersFactory.java

```diff
@@ -127,6 +133,14 @@ public void destroyRunningCluster(String clusterId) {
         DockerContainersUtil.getContainers(true).filterByName(ContainerName.getContainerNamePattern(clusterId)).kill(true).remove();
     }
 
+    public MesosCluster createMesosCluster(String path) {
+        try (InputStream is = new FileInputStream(path)) {
+            return createMesosCluster(is);
+        } catch (IOException e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1166)